src/core/cgroup.c

```diff
@@ -1036,6 +1053,11 @@ static void cgroup_context_apply(
                 }
         }
 
+        if ((apply_mask & CGROUP_MASK_CPUSET) && !is_local_root) {
+                cgroup_apply_unified_cpuset(u, c->cpuset_cpus, "cpuset.cpus");
+                cgroup_apply_unified_cpuset(u, c->cpuset_mems, "cpuset.mems");
```

Comment:
I've tested it and when empty string is written it resets the kernel value so that is handled correctly.  I've just  figured out a small issue in cgroup_apply_unified_cpuset where it would fail if the temporary buf was NULL, fixed now.